integration-cli/docker_cli_plugins_logdriver_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/docker/docker/integration-cli/checker"
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) TestPluginLogDriver(c *check.C) {
+	testRequires(c, IsAmd64, DaemonIsLinux)
+
+	pluginName := "cpuguy83/docker-logdriver-test:latest"
+
+	dockerCmd(c, "plugin", "install", pluginName)
+	dockerCmd(c, "run", "--log-driver", pluginName, "--name=test", "busybox", "echo", "hello")
```

Comment:
I ran some quick perf tests on a busybox container using `regular` log driver vs `cpuguy83/docker-logdriver-test` and it shows that there's mostly a perf hit when using the log plugins. 

```
log size(bytes):    128         256             512              1024
regular:               0.406s     0.573s        0.597s          0.469s
plugin:                 0.561s    0.577s         0.555s          0.642s
```